SA/design_concept/content/results/outlook_ide.typ

```diff
@@ -1,14 +1,19 @@
 #import "../../../acronyms.typ": *
 
 = IDE for visual functional programming
-In a future project, VisualFP could be further enhanced to work like a visual IDE for Haskell.
 
-Besides the functionality that already exists in the #ac("PoC") application, this would require additional features offered by Haskell to be implemented in a visual fashion as well,
-such as an editor for Haskell-type definitions, a way to organize user definitions into modules, integration with a package manager, etc.
+VisualFP could be further enhanced in a future project into a visual IDE for
+Haskell.
 
-On the other hand, such a development would offer the opportunity to outsource a lot of functionality, such as compilation and execution, to Haskell tooling.
-An integration with e.g. the Haskell Substitution Stepper #footnote("https://eprints.ost.ch/id/eprint/991/") could be useful.
-The already-built type-inference engine could be replaced with a Haskell compiler run as well.
+Besides the functionality already in the #ac("PoC") application, additional
+features offered by Haskell would also need to be implemented visually, such as
+an editor for Haskell-type definitions, a way to organize user definitions into
+modules, integration with a package manager, etc.
+
+On the other hand, such a development would offer the opportunity to outsource
+functionality, such as compilation and execution, to Haskell tooling.
+An integration with, e.g., the Haskell Substitution Stepper #footnote("https://eprints.ost.ch/id/eprint/991/") could be helpful.
```

Comment:
I'd switch the order of this and the next sentence

Reply:
Updated